From a2a0e3b9aeca6a25c9c00c483c82468a1d74c977 Mon Sep 17 00:00:00 2001
Message-Id: <a2a0e3b9aeca6a25c9c00c483c82468a1d74c977.1496746286.git.tsu.yubo@gmail.com>
In-Reply-To: <cover.1496746286.git.tsu.yubo@gmail.com>
References: <cover.1496746286.git.tsu.yubo@gmail.com>
From: Bo YU <tsu.yubo@gmail.com>
Date: Tue, 6 Jun 2017 18:14:30 +0800
Subject: [PATCH 2/4] staging: speakup: add a space around '|'

Add a space around logical symbol '|' to wipe out checkpatch check

Signed-off-by: Bo YU <tsu.yubo@gmail.com>
---
 drivers/staging/speakup/serialio.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/staging/speakup/serialio.c b/drivers/staging/speakup/serialio.c
index 8d2f7c672cc6..f38eb66943bf 100644
--- a/drivers/staging/speakup/serialio.c
+++ b/drivers/staging/speakup/serialio.c
@@ -138,7 +138,7 @@ static void start_serial_interrupt(int irq)
 	outb(UART_MCR_DTR | UART_MCR_RTS | UART_MCR_OUT2,
 	     speakup_info.port_tts + UART_MCR);
 	/* Turn on Interrupts */
-	outb(UART_IER_MSI|UART_IER_RLSI|UART_IER_RDI,
+	outb(UART_IER_MSI | UART_IER_RLSI | UART_IER_RDI,
 			speakup_info.port_tts + UART_IER);
 	inb(speakup_info.port_tts + UART_LSR);
 	inb(speakup_info.port_tts + UART_RX);
-- 
1.7.10.4

